testsuite/tests/with/equivalent/test.py

```diff
@@ -33,7 +34,7 @@ def check_equivalent(dep="", path="", url="", commit="", branch=""):
 
         # Cleanup
         os.chdir("..")
-        shutil.rmtree("xxx")
+        rm("xxx", recursive=True)
```

Comment:
Apologies; I probably should have left a note about this.

This is related to the fact that `file:///some/path` URLs are now always converted to a `/some/path` path before being passed to `git` commands (this conversion was previously applied inconsistently), which results in hardlinks in the `.git/objects/` directory.

For some reason, `shutil.rmtree` on Windows seems to use the permissions of the link's target rather than the link itself, and therefore [now fails](https://github.com/alire-project/alire/actions/runs/10921280622/job/30313003148#step:9:1470) in a couple of tests which pin a crate and then use `rmtree` to clear the cached clone.

I don't know much about the implications of using `e3.fs` instead, except that it is [already used elsewhere](https://github.com/alire-project/alire/blob/e20a5cbe0bb29c59c68361321c8e0bc04a81b377/testsuite/drivers/alr.py#L12) and doesn't appear to suffer from the same issue.

Reply:
OK, understood. I've seen that problem before.